docs/omnibox/index.mdx

```diff
@@ -0,0 +1 @@
+# Omnibox
```

Comment:
remove?

Reply:
@MaedahBatool this is your call

Reply:
@jdorfman yes we need to remove this. I think this is left by mistake.